pybettor/hold_calc.py

```diff
@@ -0,0 +1,27 @@
+from typing import Union  # noqa: F401
+from .implied_prob import implied_prob
+
+
+def hold_calc(odds, category: str = "us") -> float:
+    """Sportsbook Hold Calculator
+    This function calculates the hold perrcentage that the sportsbook has for the given bet.
+
+    Args:
+        odds int, float, list): odds of an event
+        category (str, optional): type of odds. Defaults to "us". \n
+            'us', American Odds \n
+            'dec', Decimal Odds \n
+            'frac', Fractional Odds
+
+    Returns:
+        float: hold percentage of odds
+    """
+
+    # Error handling
+    if not all(isinstance(line, (int, float)) for line in odds):
+        raise ValueError("Lines must be numeric")
+
+    imp_probs = implied_prob(odds, category=category)
```

Comment:
It would be helpful to be able to use the of the new methods here by passing in the method name from hold_calc.

Reply:
good callout, will make the changes in all three you tagged

Reply:
I added what I think you are looking for. Now it returns the margin from implied_prob as the hold